src/servers/daily-balance-notification.ts

```diff
@@ -1,40 +1,13 @@
-import { getRecentlyActiveAccounts } from "@app/accounts/active-accounts"
-import { getCurrentPrice } from "@app/prices"
-import { getBalanceForWallet } from "@app/wallets"
+import { sendBalanceToAccounts } from "@app/accounts/send-balance-to-users"
```

Comment:
this should be called direclty from "@app" (this is a call in a server). you should add send-balance-to-users to account/index